extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/PersistentLoginManager.java

```diff
@@ -74,14 +74,27 @@ public RestoreResult restore(RoutingContext context) {
             cipher.init(Cipher.DECRYPT_MODE, secretKey, new GCMParameterSpec(ENC_TAG_LENGTH, iv));
             String result = new String(cipher.doFinal(encrypted), StandardCharsets.UTF_8);
             int sep = result.indexOf(":");
+            // If parsing fails, something is wrong and we need to enforce a new login.
             if (sep == -1) {
+                // Enforce new login.
+                // TODO: Do we log anything? Potentially flooding the log by bots...
+                log.warnf("%s cookie parsing failed. Is encryption-key set for all instances?", cookieName);
                 return null;
             }
-            long expire = Long.parseLong(result.substring(0, sep));
-            if (System.currentTimeMillis() > expire) {
+            long expireIdle = Long.parseLong(result.substring(0, sep));
+            long now = System.currentTimeMillis();
+            log.debugf("Current time: %s", new Date(now).toString());
+            log.debugf("Expire idle timeout: %s", new Date(expireIdle).toString());
```

Comment:
There should only be a single debug line generated, flooding the logs with multiple messages makes it much harder to see what is going especially if there are multiple concurrent requests.

Reply:
Addressed.